test/org/sagebionetworks/bridge/dynamodb/DynamoSchedulePlanDaoTest.java

```diff
@@ -71,6 +71,7 @@ public void canSerializeAndDeserializeSchedulePlan() throws Exception {
         String output = mapping.writeValueAsString(abPlan);
         
         SchedulePlan newPlan = mapping.readValue(output, SchedulePlan.class);
+        newPlan.setStudyKey(abPlan.getStudyKey()); // not serialized
         
         assertEquals("Schedule plans are equal", abPlan.hashCode(), newPlan.hashCode());
```

Comment:
Minor: Can we use equals() instead of hashCode()? If equals() and hashCode() are implemented correctly (which we test for in DynamoSchedulePlanTest), then equals() and hashCode() should be equivalent, and equals() is a stronger test.